app/lib/activitypub/adapter.rb

```diff
@@ -17,6 +17,8 @@ class ActivityPub::Adapter < ActiveModelSerializers::Adapter::Base
         'conversation'              => 'ostatus:conversation',
         'toot'                      => 'http://joinmastodon.org/ns#',
         'Emoji'                     => 'toot:Emoji',
+        'cc'                        => 'https://creativecommons.org/ns#',
+        'license'                   => 'cc:license',
```

Comment:
The license term doesn't exist in cc: (it's in fact the one thing not in cc: in ccRel), it's in the xhtml namespace.  Since you probably only need one term from xhtml I'd simply do:

```
 'license'       => 'http://www.w3.org/1999/xhtml/vocab#license'
```